packages/blade/src/components/DatePicker/DateInput.web.tsx

```diff
@@ -109,7 +110,7 @@ const _DatePickerInput = (
   if (selectionType == 'single') {
     const dateValue = getFormattedDate({
       date,
-      format,
+      format: dateFormat,
```

Comment:
Can we just call this `format`?

<DatePicker dateFormat="" /> `date` sounds a bit redundant

Reply:
sure